### DISCUSSION TITLE
WATcloud Maintenance Announcement: Scheduled power outage (and other outages over 2024-2025 winter break)

### DISCUSSION THREAD

**Discussion by Jimmyj30** [WATonomous/infrastructure-support#65] [Maintenance Announcements]
### Intro

Hi @WATonomous/watcloud-compute-cluster-users, This is a WATcloud maintenance announcement. Please see below for details.

### Machines Affected

All machines

### Start Time

~~2024-12-23, 8:00 AM ET~~

2025-01-03, 8:00 AM ET

### Duration

~24-48 hours

The outage may last longer, from 2024-12-23 (or the actual outage start time) to 2025-01-10. This is because no WATcloud team members will be on campus during the winter break if we need to manually reboot a server.

### Purpose

There will be a scheduled power outage affecting the server cluster on Monday, December 23, 2024. Please plan your workloads accordingly.

### Additional Information

User VMs may be shut off on the evening before the outage to ensure that no user workloads will be impacted by unexpected power loss during the outage itself.

Enjoy the holidays – even the servers are taking a break!

**Comment by Jimmyj30** [comment]
The outage has been rescheduled by plantops to January 3, 2025: https://plantops.uwaterloo.ca/service-interruptions/notice.php?ID=2478. The rest of the announcement remains the same. Thanks for your patience!

**Comment by Jimmyj30** [comment]
Hi @WATonomous/watcloud-compute-cluster-users,

There was an outage of wato-delta (and delta-ubuntu2) around 10:22 PM ET on Nov 29, 2024. The rest of the cluster should not be significantly affected, though there might be some slowness with the Ceph-based home directory (~). Please note that the announcement for the planned power outage remains unchanged.

Thank you for your patience,
WATcloud

**Comment by jessicayuan1** [comment]
How come there’s so many outages

Just curious
Jessica

**Reply by ben-z** [reply]
We move fast and break things ;)

Just kidding, for this thread specifically, the power outage got rescheduled so the original outage didn’t happen. The outage on delta was unexpected and we have yet to find the reason. I’m suspecting some kind of overload.

In general, there are about 3 power outages per term and they result in complete cluster downtime—-unfortunately the university schedules these and we don’t yet have UPSes or other methods of protecting against these. A few more outages happen due to overloading, we are working to migrating machines into the SLURM cluster to alleviate most causes of this issue. The remaining outages are often due to human errors, which can happen a few times a term as we upgrade the cluster or deploy changes. We are constantly trying to keep this low as well.
